.github/CODEOWNERS

```diff
@@ -56,7 +56,7 @@
 /sdk/resourcemanager/         @ArcturusZhang @lirenhe @tadelesh
 
 # PRLabel: %Storage
-/sdk/storage/                 @mohsha-msft @zezha-msft @jhendrixMSFT
+/sdk/storage/                 @siminsavani-msft @souravgupta-msft @jhendrixMSFT
```

Comment:
Could we also add tasherif-msft here please?

Reply:
Yes, is there a preferred order?

Reply:
Nope, there is no preference!

Reply:
done